.github/workflows/countline.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+import os
+import sys
+
+#get current working dir
+path = os.getcwd()
+
+#parses through files and saves to a dict
+fileNamesWithSize={}
+
+#libPath and testPath dir location
+libPath = os.path.expanduser(os.path.join(path, 'lib'))
+testPath = os.path.expanduser(os.path.join(path, 'test'))
+
+#counting lines in lib dir
+for root, dirs, files in os.walk(libPath, topdown=False):
```

Comment:
`dirs` variable isn't used. Use `_` instead, which is meets the pylint, pycodestyle guidelines.